breathe/apidoc.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+"""
+    breathe.apidoc
+    ~~~~~~~~~~~~~~
+
+    Parses a directory tree looking for C/C++ modules and packages and creates
+    ReST files appropriately to create code documentation with Sphinx.  It also
+    creates a modules index (named modules.<suffix>).
```

Comment:
The comment above doesn't look right, because the script processes Doxygen XML files rather than C++ code. Also I don't see any file named `modules.<suffix>` created.